rpc/backend/call_tx.go

```
@@ -183,8 +183,8 @@ func (b *Backend) SetTxDefaults(args evmtypes.TransactionArgs) (evmtypes.Transac
 		return args, errors.New("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified")
 	}
 
-	head := b.CurrentHeader()
-	if head == nil {
+	head, err := b.CurrentHeader()
+	if head == nil || err != nil {
```

shall we return the error content?